cppcache/integration-test/ThinClientFailoverInterest.hpp

```diff
@@ -290,7 +290,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3", "Value-4"};
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```

Comment:
Above, you changed this to just a static string.  Why leave it an array here?  Let's get rid of "not-used," unless of course it _is_ used, in which case "not-used" is a bad name.

Reply:
Yup. Missed that one.